poly-synth/src/poly-synth.js

```diff
@@ -32,15 +32,38 @@ class PolySynth {
     this.releasedVoices_ = [];
 
     // The maximum and minimum cutoffs are experimentally determined.
-    this.lowPassMaxCutoff = 16000;
-    this.lowPassMinCutoff = 60;
+    this.maxCutoff = 16000;
+    this.minCutoff = 60;
 
+    // The maximum and minimum ADSR values refer to the ADSR for the gain as
+    // well as the ADSR for the filter. These values are experimentally
+    // determined.
+    this.maxAttack = 10;
+    this.minAttack = 0;
+    this.minSustain = 0.1;
+    this.maxSustain = 1;
+    this.minDecay = 0;
+    this.maxDecay = 10;
+    this.minRelease = 0;
+    this.maxRelease = 10;
```

Comment:
These are the values I've seen in Logic Pro, but Logic Pro enables nonlinear adjustment for these parameters. I can see argument for making them shorter if we don't support logarithmic control. 

At smaller values for (attack, release, and decay), these properties allow different timbral qualities, which are interesting for melodies and rhythms etc. At larger values, these controls allow the musician to create more long term movements of a song, e.g. a long filter sweep drowning out the other parts of a song, bringing the song into a different chapter.